docs/docusaurus/docs/reference/learn/gx_in_your_data_pipeline/gx_in_your_data_pipeline_lp.md

```diff
@@ -0,0 +1,27 @@
+---
+sidebar_label: 'GX in your data pipeline'
+title: 'GX in your data pipeline'
+hide_title: true
+description: Learn where GX can be integrated into a data pipeline to manage and monitor data quality.
+hide_feedback_survey: true
+pagination_next: null
+pagination_prev: null
+---
+
+import LinkCardGrid from '@site/src/components/LinkCardGrid';
+import LinkCard from '@site/src/components/LinkCard';
+import OverviewCard from '@site/src/components/OverviewCard';
+
+<OverviewCard title={frontMatter.title}>
+  Learn how GX can be integrated into a data pipeline to manage and monitor data quality.
```

Comment:
Thanks for changing "how" to "where" in the page description. Sorry that I didn't notice in the first round that this language was in two places


```suggestion
  Learn where GX can be integrated into a data pipeline to manage and monitor data quality.
```